services/movie_session.py

```diff
@@ -0,0 +1,49 @@
+from datetime import datetime
+
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+
+    cinema_hall_ = CinemaHall.objects.get(id=cinema_hall_id)
+    movie_ = Movie.objects.get(id=movie_id)
+    return MovieSession.objects.create(show_time=movie_show_time,
+                                       movie=movie_,
+                                       cinema_hall=cinema_hall_)

```

Comment:
You don't need to load db so many times here. You can use just ids to create "session".